php-practice_2.php

```diff
@@ -1,15 +1,123 @@
 <?php
 // Q1 tic-tac問題
 
+for ($calc = 1; $calc <= 100; $calc++) {
+  if ($calc % 4 === 0) {
+        echo 'tic' . "\n";
+    } elseif ($calc % 5 === 0) {
+        echo 'tac' . "\n";
+    } elseif ( $calc % 5 === 0 && $calc % 4 === 0) {
+        echo 'tic-tac' . "\n";
+    } else {
+        echo $calc . "\n";
+  }
+}
+
+
 
 // Q2 多次元連想配列
+//問題１
+$personalInfos = [
+  [
+      'name' => 'Aさん',
+      'mail' => 'aaa@mail.com',
+      'tel'  => '09011112222'
+  ],
+  [
+      'name' => 'Bさん',
+      'mail' => 'bbb@mail.com',
+      'tel'  => '08033334444'
+  ],
+  [
+      'name' => 'Cさん',
+      'mail' => 'ccc@mail.com',
+      'tel'  => '09055556666'
+  ],
+];
+
+
+echo ($personalInfos[1]['name']) . 'の電話番号は' . ($personalInfos[1]['tel']) . 'です。';
+
+//問題２
+foreach ($personalInfos as $index => $value)
+{
```

Comment:
`function`の時は改行しましたが、`foreac`などの構文の場合は改行必要ないです！
```suggestion
foreach ($personalInfos as $index => $value) {
```

Reply:
改行の位置を見直して修正しました。